Update readme with a quick start (#4)
simolus3/sqlite3.dart: README.md

In can be used both in Flutter apps or in standalone Dart applications.

The `sqlite3_flutter_libs` and `sqlcipher_flutter_libs` packages contain no Dart code at all. Flutter users can depend
on one of them to include native libraries in their apps.

## Quick Start (pure Dart)

> ```

Reviewer: For better syntax highlighting, let's make this
```suggestion
```dart
```

final db = sqlite3.openInMemory();

// Create a table.
String createTableStatement = '''
  CREATE TABLE dog (
    dog_id INTEGER PRIMARY KEY NOT NULL,
    name TEXT NOT NULL,
    breed TEXT NOT NULL
  );
''';
db.execute(createTableStatement);

> // Insert a row.
> String insertRowStatement = '''
>   INSERT INTO dog (name, breed)
>   VALUES ("Doggo", "Golden Retriever");
> ''';
> db.execute(insertRowStatement);

Reviewer: Maybe we can use

```suggestion
// Use a prepared statement to insert a row with values
PreparedStatement insertRowStatement = db.prepare('''
  INSERT INTO dog (name, breed)
  VALUES (?, ?);
''');
insertRowStatement.execute(["Doggo", "Golden Retriever"]);
```

Then we have an example for prepared statements too :)

// Get rows from table.
ResultSet resultSet = db.select("SELECT * FROM dog;");
print(resultSet);

// Update a row.
String updateRowStatement = '''
  UPDATE dog
  SET name = "Doggie", breed = "Chihuahua"
  WHERE dog_id = 1;
''';
db.execute(updateRowStatement);

resultSet = db.select("SELECT * FROM dog;");
print(resultSet);

// Delete a row.
String deleteRowStatement = '''
  DELETE FROM dog
  WHERE dog_id = 1; 
''';
db.execute(deleteRowStatement);

resultSet = db.select("SELECT * FROM dog;");
print(resultSet);

// Insert multiple rows.

> String insertRowsStatement = '''
>   INSERT INTO dog (name, breed)
>   VALUES
>     ("Doggo", "Golden Retriever"),
>     ("Doggie", "Chihuahua"),
>     ("Doggy", "Dalmatian");
> ''';
> db.execute(insertRowsStatement);

Reviewer: If we replace the first insert with a `PreparedStatement`, maybe we can use it here too:

```suggestion
insertRowStatement
  ..execute(["Doggo", "Golden Retriever"])
  ..execute(["Doggie", "Chihuahua"])
  ..execute(["Doggy", "Dalmatian"]);

insertRowStatement.dispose();
```

resultSet = db.select("SELECT * FROM dog;");
print(resultSet);

// Use the result set and iterate through the result rows.

> Iterator<Row> resultSetIterator = resultSet.iterator;
> while (resultSetIterator.moveNext()) {
>   Row row = resultSetIterator.current;
>   print('Dog Object: [id: ${row['dog_id']}, name: ${row['name']}, breed: ${row['breed']}]');
> }

Reviewer: Typically we use a `for` loop to implicitly create an iterator:
```suggestion
for (Row row in resultSet) {
  print('Dog Object: [id: ${row['dog_id']}, name: ${row['name']}, breed: ${row['breed']}]');
}
```
